recipes/urdfdom_headers/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "urdfdom_headers" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ros/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 2b3040a5f4d1e421b32d80540dd1d09fa0ef46c1d4152210ca8753c462b90e31
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+  run:
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - pkg-config --modversion {{ name }}
+    - ls $PREFIX/include/urdf_{model,sensor,world}/types.h
+
+about:
+  home: http://github.com/ros/{{ name }}
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Headers for URDF parsers'
+  dev_url: https://github.com/ros/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - jcarpent
```

Comment:
@jcarpent , could you please confirm that you also want to be a maintainer of this package on ``conda-forge`` ?

Reply:
He agree in https://github.com/conda-forge/staged-recipes/pull/8911#issuecomment-514531321